datatypes/src/error.rs

```diff
@@ -219,10 +219,7 @@ pub enum Error {
     NoAreaOfUseDefined {
         proj_string: String,
     },
-    BboxesDoNotIntersect {
-        bbox_a: BoundingBox2D,
-        bbox_b: BoundingBox2D,
-    },
+    SpatialBoundsDoNotIntersect,
```

Comment:
i think it would be nice to know which boxes did not intersect

Reply:
I agree, but the method where this error occurs is generic, I only know that it is something that impls AxisAlignedRectangle and this trait cannot be made into an object so I cannot just box it here. I could enhance the trait with a to_bbox method and always put a BoundingBox2D in the error even if it is actually a SpatialPartition2D

Reply:
works for me. lets do that

Reply:
e1409767f7d3ce765dc1529071554721b50af98b